src/pool/srv_target.c

```diff
@@ -1306,10 +1307,34 @@ ds_pool_tgt_query_aggregator(crt_rpc_t *source, crt_rpc_t *result, void *priv)
 	return 0;
 }
 
+static int
+update_vos_prop_on_targets(void *in)
+{
+	struct ds_pool 			*pool = (struct ds_pool *)in;
+	struct ds_pool_child 		*child = NULL;
+	//vos_ctl_set_policy_param 	param;
```

Comment:
???

Reply:
removed, not used, as policy_desc_t is now used